DigitalLearningSolutions.Data/Services/TrackerActionService.cs

```diff
@@ -51,5 +67,34 @@ public TrackerActionService(ITutorialContentDataService tutorialContentDataServi
 
             return ccObjectives.Any() ? new TrackerObjectiveArrayCc(ccObjectives) : null;
         }
+
+        public TrackerEndpointResponse StoreDiagnosticJson(int? progressId, string? diagnosticOutcome)
+        {
+            if (!progressId.HasValue || string.IsNullOrEmpty(diagnosticOutcome))
+            {
+                return TrackerEndpointResponse.StoreDiagnosticScoreException;
+            }
+
+            var json = JsonConvert.DeserializeObject<IEnumerable<JObject>>(diagnosticOutcome);

```

Comment:
What happens if the json string is something invalid here?

Reply:
It returns TrackerEndpointResponse.UnexpectedException, which is [#Result:-1]. Do you think it should be something else?

Reply:
It depends on the interpretation of the final sentence of the ticket: If there is an exception then log it and return "[#Result:-25]" else return "[#Result:1]". I scoped the ticket originally so I'll go and have a look what should happen.